tests/php/test_deprecation.php

```diff
@@ -70,16 +70,242 @@ function provider_deprecated_method_stubs() {
 
 	function provider_deprecated_file_paths() {
 		return array(
+			/*
 			array(
 				'class.jetpack-options.php',
 				'packages/options/legacy/class.jetpack-options.php',
 				E_USER_NOTICE,
 			),
+			*/
 			array(
 				'class.jetpack-client.php',
 				'packages/connection/src/Client.php',
 				E_USER_NOTICE,
 			),
+			array(
+				'_inc/lib/tracks/class.tracks-client.php',
+				'',
+				E_USER_NOTICE,
+			),
+			array(
+				'_inc/lib/tracks/class.tracks-event.php',
+				'',
+				E_USER_NOTICE,
+			),
+			array(
+				'class.jetpack-client.php',
+				'',
+				E_USER_NOTICE,
+			),
+			array(
+				'class.jetpack-constants.php',
+				'',
+				E_USER_NOTICE,
+			),
+			array(
+				'class.jetpack-jitm.php',
+				'',
+				E_USER_NOTICE,
+			),
+			/* Skipping because we have a global Jetpack_Options class
```

Comment:
we are adding these files back so if some third party requires one of these file it doesn't fatal